src/Resources/contao/classes/Backend.php

```diff
@@ -413,7 +413,7 @@ protected function getBackendModule($module)
 		// Custom action (if key is not defined in config.php the default action will be called)
 		elseif (\Input::get('key') && isset($arrModule[\Input::get('key')]))
 		{
-			$objCallback = new $arrModule[\Input::get('key')][0]();
+			$objCallback = System::importStatic($arrModule[\Input::get('key')][0]);
```

Comment:
Looks like a duplicate of https://github.com/contao/core-bundle/pull/435 and should be removed once that's merged.